cmd/kusk/internal/overlays/overlays_test.go

```diff
@@ -0,0 +1,12 @@
+package overlays
+
+import (
+	"testing"
+)
+
+func TestOverlay(t *testing.T) {
+	if _, err := applyOverlay("overlay.yaml", ""); err != nil {
```

Comment:
If we're adding tests here, please use Table driven tests and include a case where an error should be returned

Reply:
It will only fail if Overlay is misconfigured and I didn't want to pollute directory with more stuff.

Reply:
So in that case then, this test has no value.

Reply:
I just cleaned up the tests. Can you please take another look